src/transaction/log_page_buffer.c

```diff
@@ -1293,18 +879,14 @@ static bool
 logpb_is_dirty (THREAD_ENTRY * thread_p, LOG_PAGE * log_pgptr)
 {
   LOG_BUFFER *bufptr;		/* Log buffer associated with given page */
-  int rv;
   bool is_dirty;
 
   /* Get the address of the buffer from the page. */
-  bufptr = LOG_GET_LOG_BUFFER_PTR (log_pgptr);
-
-  START_EXCLUSIVE_ACCESS_LOG_PB (rv, thread_p);
 
+  assert (LOG_CS_OWN_WRITE_MODE (thread_p));
+  bufptr = logpb_get_log_buffer (log_pgptr);
```

Comment:
@funkydvd 

I also hope to have a blank line to separate assert for readability.